pkg/helm/controller/reconcile.go

```diff
@@ -240,7 +240,8 @@ func (r HelmOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.
 			r.EventRecorder.Eventf(o, "Warning", "OverrideValuesInUse",
 				"Chart value %q overridden to %q by operator's watches.yaml", k, v)
 		}
-		previousRelease, updatedRelease, err := manager.UpdateRelease(context.TODO())
+		force := o.GetAnnotations()["helm.sdk.operator-framework.io/upgrade-force"] == "true"
+		previousRelease, updatedRelease, err := manager.UpdateRelease(context.TODO(), release.ForceUpdate(force))
 		if err != nil {
```

Comment:
Here we need to: 
- Create a cont var to have the value of the annotation.
```
cont HelmUpgradeForceAnnotatiion = "helm.operator-sdk/upgrade-force"
```

- Use the const here instead of the String
- We should convert the string to bool value and then check it instead of comparing a string. Otherwise, it will not work if the user passes True or true for example.